jekyll/_cci2/concepts.md

```diff
@@ -19,12 +19,11 @@ This guide introduces some basic concepts to help you understand how CircleCI ma
 ## Projects
 {: #projects }
 
-A CircleCI project shares the name of the associated code repository in your VCS (GitHub or Bitbucket). Select Add Project from the CircleCI application to enter the Projects dashboard, from where you can set up and follow the projects you have access to.
+A CircleCI project shares the name of the associated code repository in your Version Control System (GitHub or Bitbucket). Select Add Project from the CircleCI application to enter the Projects dashboard, from where you can set up and follow the projects you have access to.
```

Comment:
Should we specify GH and BB, considering we are actively trying to include GitLab, and potentially others in the future? It's not a huge deal, but it would future proof the docs a little more if we stopped specifying specific VCSs.

Reply:
Do we have a page somewhere that lists supported VCSs? If we do, we can link to that instead of putting the ones we currently support in parentheses in the docs. Otherwise we will have to update every instance we see this when GitLab is fully supported.

Reply:
Yes good idea, lets remove where we can. The supported VCSs are listed multiple times here: https://circleci.com/product/

Reply:
Thank you, both! I've linked _Virtual Control System_ to https://circleci.com/docs/2.0/gh-bb-integration/ - I'm guessing that'll be updated to include GitLab in due course? Or would it be better to link to the Product page?